boilerplate/index.ts

```diff
@@ -596,8 +612,26 @@ export function createBoilerplate(sdk: CDP) {
                                     configValues: boilerplateCloudStorageApplication.configValues,
                                     enabled: true
                                 };
-                                remoteCloudStorageApplication = await bOps.applications.create(cloudStoragePayload).then(app => bOps.applications.for(app.id).get());
+                                remoteCloudStorageApplication = await bOps.applications.create(cloudStoragePayload).then(app => {
+                                    if (app.name !== "Microsoft Azure Blob") {
+                                        bOps.applications.for(app.id).auth.create({
```

Comment:
why is setting the application auth is required for the boilerplate?

Reply:
email Michael